config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false
```

Comment:
This should be removed. I've used it to have Rails return the reals errors with curl instead of the stacktrace and all of that